common-docs/offline.md

```diff
@@ -18,6 +18,26 @@ When the editor loads in your browser it tells it to keep the MakeCode service f
 
 When running on actual hardware, the code you write in JavaScript is compiled into instructions which are understood by the processor on the board. Some of the files saved in the application cache are compiled code for the features on the board. To use the hardware features on your board, the editor provides blocks for them that you include in your programs. These blocks are available to your program when they're included as part of an added extension. The code in an extension is in TypeScript and sometimes also in C++. It's necessary to compile this code at least once on the server and then have the compiled files returned to the browser to get cached. Once cached, the editor can use them together with your code, even when offline. This one time compile process is called cloud compilation.
 
+## Extensions #extensions
+
+Extensions are, by default, downloaded from internet so typically need to be connected to use them. If you want to use a particular extension **offline**, you will need to follow these steps:
+
+* **while connected to internet**, create a new program in the editor and add the extension you need to use
+* click ``|Save|`` and store the downloaded file.
```

Comment:
Which 'save'? project save, explorer save?

Reply:
The disk icon; then can use "download" too.